lib/rucio/tests/temp_factories.py

```diff
@@ -108,27 +111,32 @@ def __cleanup_rses(self):
             # So running test in parallel results in some tests failing on foreign key errors.
             rse_core.del_rse(rse_id)
 
-    def _make_rse(self, scheme, protocol_impl, add_rse_kwargs):
+    def _make_rse(self, scheme, protocol_impl, parameters={}, add_rse_kwargs={}):
```

Comment:
I advice against using mutable values as default arguments

Reply:
I initially did it on purpose, because these default arguments are rapidly dereferenced. But you are right, better do it correctly. Updated the PR